edit/buildifier.go

```diff
@@ -0,0 +1,44 @@
+package edit
```

Comment:
I suggest renaming the file to `default_buildifier.go`, otherwise it's ambiguous, people may think it's the canonical file for the buildifier binary.

Reply:
Done.